pc/src/main.rs

```diff
@@ -110,6 +93,45 @@ impl GameState {
 
 async fn run_game() {
 	let mut game_state = GameState::new();
+	let mut events = game_state.input_players.get_events();
+	let mut no_player = true;
+
+	match &mut game_state.game_run {
+		None => todo!("no game run"),
+		Some(ref mut game_run) => {
+			while no_player {
+				events = game_state.input_players.get_events();
```

Comment:
This does not check if a player is connected. It does check if an already connected players has send an event.
Ignore the return value and check if  one entry of `.player` get `Some`.